charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 1.4.0
+appVersion: 1.5.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
 version: 2.4.1
```

Comment:
should also bump this to 2.5.0 and update chart changelog

Reply:
This is already bumped by v2.5.0 via post-release. May reversed by merge. Update done know.